### PR TITLE
docker file / compose and env example

### PR DESCRIPTION
**Pull request by Sarahtonein**
Added build to compose file.
Tested in putting_it_together branch, builds and launches successuflly.

### CODE REVIEW

File: .env.example
Comment (Balugrizzly) [inline .env.example:1]: Maybe these are out of date?

**Review by Balugrizzly — APPROVED** [review]
As said in the comment for .env.example perhaps some of those envs are out of date if i recall corretly but i gonna merge now
if they they can be fixed subsequently

**Comment by Sarahtonein** [comment]
> As said in the comment for .env.example perhaps some of those envs are out of date if i recall corretly but i gonna merge now if they they can be fixed subsequently

Yep some of these are out of date and will be updated.

**Comment by Balugrizzly** [comment]
Okey will mark this here then as done

**Pull request merged**